### PR TITLE
add deploy doc

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/hosting_guide/deploy.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+Orchestra deploy on integration environment
+===========================================
+
+This tutorial is going to describe step by step how you can deploy the openOrchestra
```
Comment (itkg-rboessel) [inline en/hosting_guide/deploy.rst]: Open Orchestra

File: en/hosting_guide/deploy.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+project on the integration environment:
+
+Install Gem and Bundler
+-----------------------
+
+In order to separate all the project of your computer, we use bundler to execute only
+the specified version of capifony and capistrano
```
Comment (itkg-rboessel) [inline en/hosting_guide/deploy.rst]: link to bundler, capifony and capistrano website can be interesting

Comment (juchi) [inline en/hosting_guide/deploy.rst]: I thought capifony was dumped ?

File: en/hosting_guide/deploy.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+.. code-block:: bash
+
+    $ bundle install
+
+SSH configuration
+-----------------
+
+Make sur that your ssh configuration is ready and your personnal key is installed on
+the integration server
```
Comment (juchi) [inline en/hosting_guide/deploy.rst:31]: sure, public key

**Comment by juchi** [comment]
There are no period at the end of any sentence.

File: en/hosting_guide/deploy.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+.. code-block:: bash
+
+    $ ./composer.phar update
+    $ git add composer.lock
+    $ git commit -m "update vendor"
+    $ git pull --rebase origin master
+    $ git push origin master:update_vendor
+
+Once the tests are green, you can merge the branch into master
```
Comment (juchi) [inline en/hosting_guide/deploy.rst]: this is specific to the orchestra process (everybody don't do tests) and is not directly related to the deploy process

File: en/hosting_guide/deploy.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+Then you can run the deploy command in the project directory
+
+.. code-block:: bash
+
+    $ cap inte deploy
+
+Once this is done, you can check the project.
```
Comment (juchi) [inline en/hosting_guide/deploy.rst]: i'd rather say "the delivery is complete" or "your server has been updated". "Check" is not well suited imho

File: en/hosting_guide/deploy.rst [hunk 6]
```diff
@@ -0,0 +1,75 @@
+Orchestra deploy on integration environment
```
Comment (ngilain) [inline en/hosting_guide/deploy.rst]: deployment

File: en/hosting_guide/deploy.rst (hunk 2 above)
Comment (ngilain) [inline en/hosting_guide/deploy.rst]: I'm not sure about what "In order to separate all the project of your computer" means

**Comment by nicolasThal** [comment]
@itkg-ngilain @itkg-rboessel @juchi deploy doc updated

File: en/hosting_guide/deploy.rst [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+In the root directory of your project, run the command
+
+.. code-block:: bash
+
+    $ bundle install
+
+SSH configuration
+-----------------
+
+Make sur that your ssh configuration is ready and your personnal public key is installed on
```
Comment (ngilain) [inline en/hosting_guide/deploy.rst]: Make sure

File: en/hosting_guide/deploy.rst [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+Run the following command to test the installation and connection to the server
+
+.. code-block:: bash
+
+    $ cap inte deploy:log_revision
+
+Deploy
+------
+
+Before deploying the project, make sure that the composer.lock file is up-to-date on the master branch
```
Comment (itkg-ngilain) [inline en/hosting_guide/deploy.rst]: This is still open-orchestra team oriented as only us can merge into the master

File: en/hosting_guide/deploy.rst [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+.. code-block:: bash
+
+    $ ./composer.phar update
+    $ git add composer.lock
+    $ git commit -m "update vendor"
+    $ git pull --rebase origin master
+    $ git push origin master:update_vendor
+
+Once you are ready (passing tests for instance), you can merge the branch into master
```
Comment (ngilain) [inline en/hosting_guide/deploy.rst]: This is still open-orchestra team oriented as only us can merge into the master

**Comment by nicolasThal** [comment]
@itkg-ngilain pr updated

File: en/hosting_guide/deploy.rst [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+.. code-block:: bash
+
+    $ ./composer.phar update
+    $ git add composer.lock
+    $ git commit -m "update vendor"
+    $ git pull --rebase origin master
+    $ git push origin master:update_vendor
+
+Once you are ready (passing tests for instance), you can merge in the branch you want to deploy
```
Comment (itkg-ngilain) [inline en/hosting_guide/deploy.rst]: This is still open-orchestra team oriented as only us can merge into the master

Comment (ngilain) [inline en/hosting_guide/deploy.rst:68]: How can we choose in capistrano the branch we want to use to deploy ?

File: en/hosting_guide/deploy.rst [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+Run the following command to test the installation and connection to the server
+
+.. code-block:: bash
+
+    $ cap inte deploy:log_revision
+
+Deploy
+------
+
+Before deploying the project, make sure that the composer.lock file is up-to-date on the master branch
```
Comment (ngilain) [inline en/hosting_guide/deploy.rst]: If we can choose the branch we want to deploy, why check on the master ?

**Comment by nicolasThal** [comment]
@itkg-ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**